src/app/routes/cpsAsset/getInitialData/convertToOptimoBlocks/blocks/include/index.js

```diff
@@ -95,9 +95,16 @@ const convertInclude = async (includeBlock, ...restParams) => {
   }
 
   let ampSrc;
+  let ampImage;
+  let ampImageWidth;
+  let ampImageHeight;
   let html;
   if (classification === 'vj-supports-amp') {
     ampSrc = ampSrcBuilder(href);
+    const queryParams = new URLSearchParams(href);
+    ampImageWidth = queryParams.get('amp-image-width');
+    ampImageHeight = queryParams.get('amp-image-height');
+    ampImage = queryParams.get('amp-image');
```

Comment:
I'd consider breaking this logic out into a seperate utility or maybe expanding the scope of `ampSrcBuilder` so it extracts all this other metadata too, so maybe:
`ampMetadata = ampMetadataExtractor(href);`

then further down in the return: 
```
 return { 
    type,
    model : { 
       href,
       type: includeType,
       ...(ampMetadata ? ampMetadata : {} )
       ..
   }
```

Just once again pushes the testing further down into a smaller utility rather than just at the converter level.